### PR TITLE
Add detailed latency option and add automated scenarios

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <HtmlJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.html.json"</HtmlJobs>
     <JsonJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.json.json"</JsonJobs>
     <MultiQueryJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.multiquery.json"</MultiQueryJobs>
+    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json</SignalRJobs>
```
Comment (sebastienros) [inline build/repo.targets]: Where do you define the table to store the results?

File: build/repo.targets [hunk 2]
```diff
@@ -69,6 +70,13 @@
     <CloudScenarios Include="-n MvcDbFortunesRaw --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
     <CloudScenarios Include="-n MvcDbFortunesDapper --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
     <CloudScenarios Include="-n MvcDbFortunesEf --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
+
+    <CloudScenarios Include="-n SignalRBroadcast --properties 'TransportType=WebSockets;HubProtocol=json' $(SignalRJobs)" />
```
Comment (sebastienros) [inline build/repo.targets]: `TransportType=WebSockets;HubProtocol=json` Do you set these properties in the database as custom columns?

File: build/repo.targets (hunk 1 above)
Comment (BrennanConroy) [reply build/repo.targets]: Odd, I must have accidentally deleted it

File: build/repo.targets (hunk 2 above)
Comment (BrennanConroy) [reply build/repo.targets]: Yes

**Pull request merged**